### DISCUSSION TITLE
Error while copying the mysql repo file

### DISCUSSION THREAD

**Discussion by omdevops99** [learndevopsonline/learndevopsonline#1617] [Q&A] [answered]
Hi Team,
I'm getting the below error, while copying the mysql repo file.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/141154551/bbe15a33-2fa3-4cb2-b999-494dacf134cf)
workstation ip: 107.23.234.27

**Comment by sreedharm07** [comment]
Hi @omswetha 
remove remote_src line, if we want to copy file from remote(files in workstation) we give remote_src,
but,
we are copying file from local. so remote that line and run it will copy
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/01e13546-9841-47dd-b162-00e3231de734)

**Reply by omdevops99** [reply]
Thanks, one more issue please.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/141154551/1b33a39a-1c95-44fb-9d19-3bff3117264e)

**Reply by sreedharm07** [reply, marked as answer]
@omswetha 
syntax error please add remote_src in your code. it will work
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/39865204-75c7-4af4-88d6-5355202c339c)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/c0176aeb-3574-4846-9c0c-6d5982761ab6)

**Reply by omdevops99** [reply]
Thanks!!
